Extension/src/common.ts

```diff
@@ -79,6 +31,29 @@ export function getPackageJsonString(): string {
     return JSON.stringify(packageJson, null, 2);
 }
 
+// Extension is ready if install.lock exists and debugAdapters folder exist.
+export async function isExtensionReady(): Promise<boolean> {
+    const doesDebugAdapterFolderExist: boolean = await checkFolderExists(getExtensionFilePath("debugAdapters"));
```

Comment:
>onst doesDebugAdapterFolderExist: boolean = await checkFolderExists(getExtensionFilePath("debugAdapters")); [](start = 5, length = 107)

Publicly this will always be true. Why are we checking for it?